src/device.rs

```diff
@@ -522,8 +522,8 @@ impl Device {
     ///
     /// This is a modification only affecting this representation of
     /// this device.
-    pub fn set_clock_id(&self, clockid: i32) -> io::Result<()> {
```

Comment:
What's the reason for changing it to `c_int`? I prefer not to use libc types in the user interface.

Reply:
I agree that the change as it stands doesn't make sense. I meant to change it to `libc::clockid_t`. The reason is just that C's `clockid_t` is not guaranteed to be the same as `i32` on all platforms (for example on linux/gnu it is signed int and on dragonfly bsd it is unsigned long). This difference probably won't trigger undefined behavior, as clockids are often low positive numbers (and this library only works on linux). I understand your weariness to expose the types of dependencies, but in the case of these libc type aliases, they are both extremely stable and serve as documentation.

Whatever you say goes here.

Reply:
I would prefer sticking to i32 in this case. I am not concerned about it not being i32 on non-linux/gnu platforms. I just want to avoid using `libc` types in the user interface.

Please modify it accordingly and we can finally get this patch merged :)